university-of-lincoln-harvard.csl

```diff
@@ -223,31 +220,27 @@
           </group>
         </else-if>
         <else-if type="book graphic report" match="any">
-          <group delimiter=". " prefix=" ">
+          <group prefix=" " delimiter=". ">
             <text variable="note"/>
-            <group delimiter=", ">
-              <text macro="title"/>
-              <group delimiter=" ">
-                <text variable="collection-title"/>
-                <text variable="collection-number"/>
-              </group>
-              <text macro="edition"/>
+            <text macro="title"/>
+            <group delimiter=" ">
+              <text variable="collection-title"/>
+              <text variable="collection-number"/>
             </group>
+            <text macro="edition"/>
             <text macro="editor"/>
             <text macro="volume"/>
             <text macro="publisher"/>
           </group>
         </else-if>
         <else-if type="chapter paper-conference" match="any">
-          <group suffix=".">
-            <text macro="title" prefix=" "/>
-            <text macro="edition" prefix=", "/>
-          </group>
+          <text macro="title" prefix=" " suffix="."/>
           <group prefix=" " delimiter=" ">
             <text term="in" text-case="capitalize-first" suffix=":"/>
             <text macro="editor"/>
             <text variable="container-title" font-style="italic" suffix="."/>
             <text variable="collection-title" suffix="."/>
+            <text macro="edition" prefix=" "/>
```

Comment:
No need for the prefix given the group delimiter, though you may need a suffix to separate edition from date

Reply:
I see - I'll get rid of that and take a look at if we need the suffix.